### PR TITLE
Zkp section addition

### PR DESCRIPTION
**Pull request by brentzundel**
Clarification paragraph added to ZKP section in response to feedback.

***
<a href="https://pr-preview.s3.amazonaws.com/brentzundel/vc-data-model/pull/460.html" title="Last updated on Mar 19, 2019, 2:51 PM UTC (9420f3f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/460/9331c44...brentzundel:9420f3f.html" title="Last updated on Mar 19, 2019, 2:51 PM UTC (9420f3f)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2405,7 +2405,31 @@ <h3>Zero-Knowledge Proofs</h3>
 has granted a degree to you without revealing your identity or any other
 personally identifiable information contained on the degree.
         </p>
-
+<p>
```
Comment (davidlehn) [inline index.html]: Style nits: Properly indent this `<p>`, add  indented `</p>` below next line, indent the `<ol>`, remove `</p>` at bottom of this change.

**Comment by davidlehn** [comment]
Can you rebase or squash the PR to avoid all the extra commits?

**Comment by ken-ebert** [comment]
The contents of the text appear accurate to me.

File: index.html [hunk 2]
```diff
@@ -2405,7 +2405,31 @@ <h3>Zero-Knowledge Proofs</h3>
...
+The key capabilities introduced by zero-knowledge proof mechanisms are:
+ <ol>
+          <li>
+The ability of a <a>holder</a> to combine multiple <a>verifiable credentials</a>
+from multiple <a>issuers</a> into a single presentation without revealing
+correlating identifiers to the <a>verifier</a>. This makes it more difficult
```
Comment (Fak3) [inline index.html]: Identifiers of what?

**Comment by brentzundel** [comment]
> Can you rebase or squash the PR to avoid all the extra commits?

If others agree this is necessary, I will undergo the process, but I'd rather not have to do that. Especially because merging the PR squashes the commitments already.

**Comment by chaals** [comment]
**Personally** I'm fine with letting it get squashed on merge...

**Pull request merged**